app/stats.go

```diff
@@ -0,0 +1,26 @@
+package app
+
+import (
+	"github.com/Sirupsen/logrus"
+	"github.com/codegangsta/cli"
+)
+
+func LsStats() cli.Command {
+	return cli.Command{
+		Name:      "ls-stats",
```

Comment:
What about journal, log, or telemetry.  Seems like we will add stats later and it will be key/values, not this log data.